addons/website/static/tests/tours/snippet_popup_open_on_top.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module */
+
+import tour from "web_tour.tour";
+import wTourUtils from "website.tour_utils";
+
+function fail (errorMessage) {
+    tour._consume_tour(tour.running_tour, errorMessage);
+}
+
+wTourUtils.registerWebsitePreviewTour("snippet_popup_open_on_top", {
+    test: true,
+    url: "/",
+    edition: true,
+}, [
+    wTourUtils.dragNDrop({id: "s_popup", name: "Popup"}),
+    {
+        content: "Change content for long text.",
+        trigger: "iframe .s_popup p.lead",
+        run: 'text '+' hello world'.repeat(300),
+    },
+    {
+        content: "Set delay to 0 second",
+        trigger: '[data-attribute-name="showAfter"] input',
+        run: 'text 0',
+    },
+    ...wTourUtils.clickOnSave(),
+    {
+        content: 'Check that the modal is scrolled on top',
+        trigger: "iframe .s_popup .modal:contains('hello world')",
+        run: function () {
+            const modal = document.querySelector('iframe').contentDocument.querySelector('.s_popup .modal');
```

Comment:
- In website, we try to suffix element variable with `El` (so `modalEl`)
- `const modalEl = this.$anchor[0]` ?